portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,67 @@
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+ // generate the project contents when the project.html document is loaded
+window.onload = function generateProjectContent() {
+      const projects = {
+        "Accessible Mouse Design": ["Designed and prototyped a foot mouse with a pedal and a control base",
+                                    "Created functional code using processing and java to satisfy all basic mouse functions including clicking, scrolling, and navigation",
+                                    "Focused on accessibility and user experience",
+                                    "Project won first place in the University of Toronto Designathon League"],
+    
+        "PASS Device": ["Designed an enhancement to the **Personal Alarm Safety System (PASS)** used by Toronto firefighters",
+                                    "Created a device that prevents false alarms triggered by the existing system with a button embedded in the firefighters' gloves, which signals that they are conscious",
+                                    "New system envisioned to use infraed signals to communicate between button and device --prototype was built using an Arduino and a motor that when triggered, causes rotatable arm to deactivate PASS alarm",
+                                    "Conducted research with input from Toronto Fire Department to ensure safety standards were met"],
+        "ESEC Website": ["Created and maintained a multi-page website for the Engineering Science Education Conference using WordPress and custom html/css",
+                                    "Analyzed previous legacy code and utilized it as boilerplate infrastructure while updating source-code as required",
+                                    "Applied concepts of responsive web design in order to make page accessible and user-freindly across all devices"], 
+        "Matboard Construction Bridge Project": ["Constructed a ~1.5m long bridge using matboard only",
+                                    "Applied knowledge of stress, shear, and buckling to optimize bridge load",
+                                    "Acheived maximum load of 800N before failure"]                             
+        };
+      var TheInnerHTML ="";
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+      for (project in projects) {
+          TheInnerHTML += "<h5>"+project+"</h5>";
+          TheInnerHTML += "<ul>";
+          for (i = 0; i < projects[project].length; i++) {
+              TheInnerHTML += "<li style=\"list-style-type:circle\">"+projects[project][i]+"</li>";
+          }
+          TheInnerHTML += "</ul>";
+    }
+    document.getElementById("projectContent").innerHTML = TheInnerHTML;
+}
+
+//play a game of two truths and one lie
+function playGame() {
```

Comment:
does this mean I can cheat by opening the source in the chrome dev tools?

Reply:
Ah I didn't think of that. How can I hide the source code then? Should I disable the right click or implement an encryption method?

Reply:
Don't block this change on fixing it. It's not a big deal, and still a fun game. If it was a serious security question, I would do the following:

I would probably solve this with a request to the backend.
- have these lists stored in the backend
- have an API that gets two truths and a lie (but doesn't contain the answer)
- client calls backend, gets 3 statements
- user picks one
- client calls a different backend API to find out which is a lie

That way, nothing served to the end user has the answer accessible anywhere. It all lives on the server which the user cannot access.

Reply:
That makes sense, thank you!